positron/components/ModuleLoader.jsm

```diff
@@ -42,28 +42,32 @@ const globalPaths = [
   'node/',
 ];
 
-/**
- * Mapping from module IDs (resource: URLs) to module objects.
- *
- * @keys {string} The ID (resource: URL) for the module.
- * @values {object} An object representing the module.
- */
-let modules = new Map();
+const windowLoaders = new WeakMap();
 
 /**
  * Construct a module importer (`require()` global function).
  *
  * @param requirer {Module} the module that will use the importer.
  * @return {Function} a module importer.
  */
-function Require(requirer) {
+
+function ModuleLoader(processType) {
+  /**
+   * Mapping from module IDs (resource: URLs) to module objects.
+   *
+   * @keys {string} The ID (resource: URL) for the module.
+   * @values {object} An object representing the module.
+   */
+  let modules = new Map();
```

Comment:
> s/let/const

This change is actually part of #1, on which this branch is based. See this comparison between the two branches to get an accurate diff of the changes made in this PR:

https://github.com/mykmelez/positron/compare/renderer-process-modules...implement-process-platform

In any case, I've made this change in ac43d30 on that branch.